vpr/test/test_noc_storage.cpp

```diff
@@ -361,5 +358,48 @@ TEST_CASE("test_remove_link", "[vpr_noc]") {
         REQUIRE(test_noc.remove_link(link_to_remove_src_router, link_to_remove_sink_router) == false);
     }
 }
+TEST_CASE("test_generate_router_key_from_grid_location", "[vpr_noc]") {
+    // The golden set of router ids that constain the location of each router
```

Comment:
Can you describe exactly what the test tests, and how it works at a high level?

Reply:
Updated comment